driver.go

```diff
@@ -123,8 +123,37 @@ func (d *sqlDriver) unlock() {
 	d.mu.Unlock()
 }
 
-// Open returns a new connection to the database.  The name is a string in a
-// driver-specific format.
+// parseDriverDSN cuts off DB name from dataSource name. It returns error if
+// dataSource is not valid.
+func parseDriverDSN(dataSource string) (storeDSN, dbName string, err error) {
```

Comment:
DSN contains database name sometimes, see https://en.wikipedia.org/wiki/Data_source_name
so using storePath instead of  storeDSN may be better. 

btw, dataSource -> dsn